nestkernel/recording_backend_mpi.cpp

```diff
@@ -383,8 +385,12 @@ nest::RecordingBackendMPI::get_port( const size_t index_node, const std::string&
   }
 
   basename << "/" << index_node << ".txt";
-  std::cout << basename.rdbuf() << std::endl;
+  std::cout << basename.str() << std::endl;
   std::ifstream file( basename.str() );
+  if ( !file.good() )
+  {
+    throw MPIPortsFileMissing( index_node );
```

Comment:
Wouldn't it be nicer to pass both the expected file name and the node_id here and then print a message like this?
```
Node <node_id> expects a file with the MPI address at location <file_name>. This does not seem to exist.
```